dev-environment/docker-compose.yaml

```diff
@@ -51,6 +51,15 @@ services:
     networks:
       - mlflow-net
 
+  pmml-samples:
+    build: pmml-samples/
+    environment:
+      - PMML_SERVER_PORT=${PMML_SERVER_PORT}
+    volumes:
+      - ./pmml-samples/:/home/pmml
+    networks:
+      - pmml-net
```

Comment:
I'm not really familiar with how networks work in Docker, but do we need a new network for each machine?

Reply:
it semantically should be separate network as this part of env has nothig common with mlflow

Reply:
don't we want another docker-compose for pmml? we don't need to load mlflow, while testing pmml so this can reduce some time